pytorch_lightning/utilities/device_parser.py

```diff
@@ -74,13 +80,19 @@ def parse_gpu_ids(gpus: Optional[Union[int, str, List[int]]]) -> Optional[List[i
             indicates specific GPUs to use.
             An int 0 means that no GPUs should be used.
             Any int N > 0 indicates that GPUs [0..N) should be used.
+        include_cuda: A boolean indicating whether to include cuda devices for gpu parsing.
+        include_mps: A boolean indicating whether to include mps devices for gpu parsing.
```

Comment:
I didn't understand where a user would set these. Is this just for testing?

Reply:
the user never uses these. They are just to unify logic between cuda/gpu and mps